tools/docker-compose.override.yml

```diff
@@ -31,7 +31,6 @@ services:
     environment:
```

Comment:
In line 18 above, it should be the case that this line can be removed:

```
      CONTROL_CENTER_ZOOKEEPER_CONNECT: 'zookeeper:2181'
```

Reply:
Great point, it has been removed in [this commit](https://github.com/confluentinc/kafka-tutorials/pull/590/commits/5d9413a0637aa1ec13ac3439a8c69709ae62257f)